src/plugins/offboard/offboard_impl.cpp

```diff
@@ -437,6 +465,46 @@ Offboard::Result OffboardImpl::send_position_velocity_ned()
                                             Offboard::Result::ConnectionError;
 }
 
+Offboard::Result OffboardImpl::send_acceleration_ned()
+{
+    const static uint16_t IGNORE_X = (1 << 0);
+    const static uint16_t IGNORE_Y = (1 << 1);
+    const static uint16_t IGNORE_Z = (1 << 2);
+    const static uint16_t IGNORE_VX = (1 << 3);
+    const static uint16_t IGNORE_VY = (1 << 4);
+    const static uint16_t IGNORE_VZ = (1 << 5);
+    const static uint16_t IGNORE_YAW_RATE = (1 << 11);
+
+    const auto acceleration_ned = [this]() {
+        std::lock_guard<std::mutex> lock(_mutex);
+        return _acceleration_ned;
+    }();
+
+    mavlink_message_t message;
+    mavlink_msg_set_position_target_local_ned_pack(
+        _parent->get_own_system_id(),
+        _parent->get_own_component_id(),
+        &message,
+        static_cast<uint32_t>(_parent->get_time().elapsed_s() * 1e3),
+        _parent->get_system_id(),
+        _parent->get_autopilot_id(),
+        MAV_FRAME_LOCAL_NED,
+        IGNORE_X | IGNORE_Y | IGNORE_Z | IGNORE_VX | IGNORE_VY | IGNORE_VZ | IGNORE_YAW_RATE,
```

Comment:
should we add `IGNORE_YAW` as well?

Reply:
aha maybe that's needed, let me do more tests